devsetup/Makefile

```diff
@@ -379,6 +380,7 @@ edpm_baremetal_compute: export OPERATOR_NAME=openstack
 edpm_baremetal_compute: export NAMESPACE=${BMH_NAMESPACE}
 edpm_baremetal_compute: export BMAAS_NETWORK_NAME=${BM_NETWORK_NAME}
 edpm_baremetal_compute: export BMAAS_INSTANCE_NAME_PREFIX=${BM_INSTANCE_NAME_PREFIX}
+edpm_baremetal_compute: export BMAAS_INSTANCE_NAME_SUFFIX=${BM_INSTANCE_NAME_SUFFIX}
```

Comment:
This sounds like the initial index used for new instances added. I assume if one has to add new instances they have to check the count of existing instances and set this accordingly which kind of not inituitive to me.

Reply:
yes, that's how it works and I added some docs for that that. It's the same way the edpm_compute targets work. I think for developers, it's ok.